testing/infrastructure/CatalogShape.ttl

```diff
@@ -45,15 +45,21 @@ shapes:ResourceCatalogShape
 	sh:property [
 		a sh:PropertyShape ;
 		sh:path ids:offeredResources ;
```

Comment:
This should be `ids:offeredResource` (-> without the "s" at the end)

Reply:
@HaydarAk can you fix this I don't have permissions to do so